src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_input.tsx

```diff
@@ -148,18 +168,34 @@ export const WorkspacePermissionSettingInput = ({
                 })
           }
           isDisabled={userOrGroupDisabled}
+          prepend={
+            <EuiFlexItem style={{ maxWidth: 100 }}>
+              <EuiSelect
+                className="workspacePermissionSettingUserAndGroupSelect"
+                compressed={true}
+                options={typeOptions}
+                onChange={handleTypeChange}
+                value={type}
+                style={{
+                  boxShadow: 'none',
+                  fontWeight: 'bold',
+                }}
```

Comment:
Is it necessary that we overwrite the style here? Can we use whatever OUI component has?

Reply:
There isn’t an oui component example using prepend to implement the dropdown selection, so I use the select component as prepend here. In the Figma design, this part has no shadow and the text is bold, but the select component does not provide APIs to control the shadow and font weight, so I overwrite the style here.

Reply:
There is an example here, would that help? https://oui.opensearch.org/1.9/#/forms/form-controls#prepend-and-append
<img width="447" alt="image" src="https://github.com/user-attachments/assets/a8abc920-b843-4f76-a166-d0d32a7bf261">

Reply:
Yes I'm trying to use popover to implement the type switch.